cpp/src/arrow/compute/exec/tpch_node_test.cc

```diff
@@ -333,24 +343,15 @@ void CountModifiedComments(const Datum& d, int* good_count, int* bad_count) {
   }
 }
 
-TEST(TpchNode, ScaleFactor) {
```

Comment:
All of these methods moving around makes it a little harder to follow changes.  Can we maybe pick an order (e.g. ScaleFactor, each table in alphabetical order, AllTables) so we can stick with it?  Or is there some meaning behind this new order?

Reply:
The order is the order in which the tables appear in the spec: Supplier, Part, Partsupp, Customer, Orders, Lineitem, Nation, Region. 
ScaleFactor was put under VerifySupplier since it was generating the Supplier table, but I've gotten rid of it.

Reply:
Great.  As long as there is some kind of order we can stick with going forwards I'm content.